packages/cli/src/utils.ts

```diff
@@ -70,16 +72,36 @@ export async function parseConfigFilename(filename: string): Promise<string> {
   return filename;
 }
 
-export function ora(opts?: realOra.Options): realOra.Ora {
-  return realOra({ stream: process.stdout, ...opts });
+export function logSuccess(message: string) {
+  spinner.succeed(message);
+}
+
+export function logFail(message: string) {
+  spinner.fail(message);
+}
+
+export function logStart(message: string) {
+  if (process.stdout.columns == 0 && process.stdout.rows == 0) {
+    console.log(message);
+  } else {
+    spinner.start(message);
+  }
+}
+
+export function logInfo(message: string) {
+  spinner.info(message);
+}
+
+export function logWarn(message: string) {
+  spinner.warn(message);
 }
 
 /**
  * leave a message and abort the process
  * @param spinner spinner object
  * @param message last message
  */
-export function abort(spinner: realOra.Ora, message: string): void {
-  spinner.fail(message);
+export function abort(message: string): void {
```

Comment:
I think it's good to add an optional parameter `code` or `codestr` to specify which code will be passed to `process.exit`.